### ISSUE TITLE
Monitoring as a daemon mode

### ISSUE THREAD

**Issue by wookayin** [wookayin/gpustat#7]
Currently, one has to use `watch --color -n0.1 gpustat` for periodical watching.
A built-in daemon mode (collecting some statistics as well) is supposed to be added.

**Comment by wookayin** [comment]
In progress: #25 #30

**Comment by wookayin** [comment]
This is now released as 0.5.0. As described in README, the following should work:

```
gpustat -i               # refresh every 1 sec (Default)
gpustat -i 0.1         # refresh every 0.1 sec
```
